elasticsearch/run.sh

```diff
@@ -60,4 +57,18 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+TIMES=60
```

Comment:
so... if recovery takes longer than 60 seconds, we bail out? or am I misreading what's below?